decidim-conferences/app/commands/decidim/conferences/join_conference.rb

```diff
@@ -51,7 +51,7 @@ def create_meetings_registrations
         meetings = Decidim::Meetings::Meeting.where(component: published_meeting_components).where(id: @registration_type.conference_meetings.pluck(:id))
 
         meetings.each do |meeting|
-          Decidim::Meetings::Registration.create!(meeting:, user:)
+          Decidim::Meetings::Registration.create!(meeting:, user: current_user)
```

Comment:
Where does this `current_user` comes from? The initializer is for `@user` and we have an `attr_reader` so with `user` is enough

Reply:
As we don't have any `form` here, this should be out of the scope of the current refactoring

Reply:
I am not sure why I called the current_user here. Perhaps as an attempt to fix a error within a previous spec with an error in the registrations. 

Thoroughly appericate the second pair of eyes!